fk7263/src/main/java/se/inera/intyg/common/fk7263/model/converter/certificate/question/QuestionArbetslivsinriktadRehabilitering.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.fk7263.model.converter.certificate.question;
+
+import static se.inera.intyg.common.fk7263.model.converter.RespConstants.ARBETSLIVSINRIKTAD_REHABILITERING_CATEGORY_ID;
+import static se.inera.intyg.common.fk7263.model.converter.RespConstants.ARBETSLIVSINRIKTAD_REHABILITERING_SVAR_ID;
+
+import se.inera.intyg.common.fk7263.model.internal.Rehabilitering;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigViewText;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueViewText;
+
+public class QuestionArbetslivsinriktadRehabilitering {
+
+    private static final String NO = "Nej";
+    private static final String YES = "Ja";
+    private static final String NOT_PROVIDED = "Ej angivet";
+    private static final String UN_JUDGEABLE = "Går inte att bedöma";
+
```

Comment:
Suggestion to remove underscore and use UNJUDGEABLE

Reply:
Done!